src/System.Threading.Tasks.Parallel/tests/ParallelFor.cs

```diff
@@ -0,0 +1,686 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using CoreFXTestLibrary;
+
+using System;
+
+namespace System.Threading.Tasks.Test.Unit
+{
+
+    public sealed class ParallelForUnitTests
+    {
+        [Fact]
+        public static void ParallelForTest0()
+        {
+            TestParameters parameters = new TestParameters(API.For64, StartIndexBase.Int32)
+            {
+                Count = 0,
```

Comment:
As we discussed offline, there's a lot of repetition in these tests that could easily be refactored into helpers.  If you're not going to address those prior to merging, could you please open an issue to make sure it gets follow-up on?